src/components/data-processing/StatusIndicator.jsx

```diff
@@ -29,13 +28,11 @@ const StatusIndicator = (props) => {
     experimentId, allSteps, currentStep, completedSteps,
   } = props;
 
-  const {
-    status: { pipeline },
```

Comment:
The `initialValue` for status is `null`, so this throws an error because we can't read `pipeline` from `undefined`. We do not want to hard code this in the UI because the `status` object is returned from the API.